commit 0076bfb94ee7212f3c4b24cf4654cc3773f1d460 (refs/changes/99/5699/2)
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2019-03-04T12:14:29-08:00 (7 months ago)
    
    OS-7618 vminfod fswatch is skipping fs events, leading to a vminfod crash
    Reviewed by: Dave Eddy <dave.eddy@joyent.com>
    Reviewed by: Mike Gerdts <mike.gerdts@joyent.com>
    Reviewed by: Orlando Vazquez <orlando@joyent.com>
    Approved by: Orlando Vazquez <orlando@joyent.com>

diff --git a/src/vm/node_modules/vminfod/vminfod.js b/src/vm/node_modules/vminfod/vminfod.js
index b5131232..2039f4c9 100644
--- a/src/vm/node_modules/vminfod/vminfod.js
+++ b/src/vm/node_modules/vminfod/vminfod.js
@@ -736,7 +736,7 @@ Vminfod.prototype.startWatchers = function startWatchers(callback) {
 
     vasync.pipeline({funcs: [
         function startFsWatcher(_, cb) {
-            self.fsw = new FsWatcher({log: self.log, dedup: true});
+            self.fsw = new FsWatcher({log: self.log});
 
             self.fsw.on('event', function fswOnEvent(obj) {
                 self.handleFsEvent(obj);
diff --git a/src/vm/package.json b/src/vm/package.json
index 648002c3..866a6685 100644
--- a/src/vm/package.json
+++ b/src/vm/package.json
@@ -1,7 +1,7 @@
 {
   "name": "vmadm",
   "description": "administrative tool(s) for managing VMs",
-  "version": "1.0.2",
+  "version": "1.0.3",
   "author": "Joyent (joyent.com)",
   "private": true,
   "devDependencies": {
